proto/src/determined/experiment/v1/searcher.proto

```diff
@@ -165,12 +165,12 @@ message ShutdownOperation {
   int32 placeholder = 1;
 }
 
-// SearcherOperation is an operation issued by the searcher.
+// SearcherOperation is an operation issued by the custom searcher.
 message SearcherOperation {
   // A searcher operation is one of the following operations.
   oneof union {
-    // ValidateAfter is issued to tell a trial to train some then validate.
-    ValidateAfterOperation validate_after = 1;
+    // TrialOperation is issued for the current trial. .
```

Comment:
I don't think this is quite what @dzhu had in mind.
Before Custom Searcher there already was something called "SearcherOperation". That something should become "TrialSearcherOperation" or "TrialOperation" if you prefer. It used to contain "ValidateAfterOperation", just like `TrialOperation` that you defined below.
However, it is _not_ that `TrialOperation` that we want to include into the new `SearcherOperation` but rather either `ValidateAfrterOperation` directly or something equivalent.